flink-runtime/src/main/java/org/apache/flink/runtime/taskexecutor/TaskManagerRunner.java

```diff
@@ -91,7 +93,8 @@
 
 	private static final int STARTUP_FAILURE_RETURN_CODE = 1;
 
-	public static final int RUNTIME_FAILURE_RETURN_CODE = 2;
+	@VisibleForTesting
```

Comment:
Why is this necessary?

Reply:
I saw this field only used in `TaskManagerRunner.java` and testing code. So I think it probably not makes sense to set the access modifier to `public`. However, we could not set it to `private` as it is used by testing code. So, I change it to `package-private` and annotate it with `VisibleForTesting`. It just a hotfix though.

Reply:
Ah ok. This makes sense. I did not see the hotfix commit.